src/index.html

```diff
@@ -394,6 +400,25 @@ <h2 class="font-weight-bold">Contact</h2>
                 </div>
             </form>
         </section>
+
+        <section class="col-12 py-5" id="board-members">
+            <div class="text-center text-black mb-5">
+                <h2 class="font-weight-bold">Board Members</h2>
+                <ul style="list-style-type: none">
+                    <li><a>President: </a><a href = "mailto:elsbeth@girlscodelincoln.org"style="color: indigo">Elsbeth Magilton</a></li>
```

Comment:
You can remove the `<a>` tags from around the titles so it would be 
`<li> President: <a href = "mailto:elsbeth@girlscodelincoln.org" style="color: indigo">Elsbeth Magilton</a></li>`